dist/localforage-removeitems.es6.js

```diff
@@ -33,30 +33,31 @@ function removeItemsIndexedDB(keys, callback) {
             var dbInfo = localforageInstance._dbInfo;
             var transaction = dbInfo.db.transaction(dbInfo.storeName, 'readwrite');
             var store = transaction.objectStore(dbInfo.storeName);
+            var lastError;
+
+            transaction.oncomplete = function () {
+                resolve();
+            };
+
+            transaction.onabort = transaction.onerror = function () {
+                reject(lastError || transaction.error || 'Unknown error');
+            };
+
+            function requestOnError(evt) {
+                var request = evt.target || this;
+                lastError = request.error || request.transaction.error;
```

Comment:
Yes something like this but with {} to the if. You are right that subsequent calls to rejectv should have no effect according to the spec but various Promise libs like bluefish, detect such out-of-spec cases on runtime and log warnings, in an effort to help identify possible issues. (I still remember getting a warning about not returning a promise that was created inside a `then()` call.)
Whatever... Even without that change, I will try to merge and push a new version tomorrow.